docs/src/using.md

```diff
@@ -10,9 +10,9 @@ julia> using CategoricalArrays
 julia> x = CategoricalArray(["Old", "Young", "Middle", "Young"], ordered=true)
 4-element CategoricalArrays.CategoricalArray{String,1,UInt32}:
  "Old"   
- "Young" 
+ "Young"
```

Comment:
Are you sure it's a good idea to remove these? As you can see, I just copy/pasted the REPL output, so we're certain the doctests will pass. I don't see the point of removing them.

Reply:
oops. It's not me it's Atom :) I didn't realize it's padding, will revert